test/cli.js

```diff
@@ -52,13 +52,26 @@ test('ignore files in .gitignore', async t => {
 	const cwd = path.join(__dirname, 'fixtures/gitignore');
 
 	try {
-		await execa('../../../cli.js', ['--no-local'], {cwd});
+		const result = await execa('../../../cli.js', ['--no-local'], {cwd});
+		console.log(result, result.stdout, result.stderr);
+		t.fail();
 	} catch (err) {
 		t.is(err.stdout.indexOf('foo.js'), -1);
 		t.true(err.stdout.indexOf('bar.js') !== -1);
 	}
 });
 
+test.failing('negative gitignores', async t => {
+	const cwd = path.join(__dirname, 'fixtures/negative-gitignore');
+
+	try {
+		await execa(`../../../cli.js`, [`${cwd}/bar.js`, '--no-local'], {cwd});
+		t.fail();
+	} catch (err) {
+		t.is(err.stdout.indexOf('foo.js'), -1, 'Should not lint foo.js');
+	}
+});
```

Comment:
This is cool! I didn't know that you could `await t.throws` and then make further assertions with the error. Thanks and I changed it!